spec/requests/api/v1/accounts_spec.rb

```diff
@@ -163,6 +163,31 @@
       end
     end
 
+    context 'when user tries to follow their own account' do
+      subject do
+        post "/api/v1/accounts/#{other_account.id}/follow", headers: headers
+      end
+
+      let(:locked) { false }
+      let(:other_account) { user.account }
+
+      it 'returns http forbidden and error message' do
+        subject
+
+        error_msg = 'Following your own account is not allowed'
+
+        expect(response).to have_http_status(403)
+
+        expect(response.parsed_body)
+          .to include(
+            error: error_msg,
+            following: false
```

Comment:
Is this param (following) actually in the json?

Reply:
No it isn't. I think it was based on the assumption that there are some additional JSON fields that we return in addition to the error message. Have removed it.